packages/css/src/components/footer/README.md

```diff
@@ -6,17 +6,17 @@ Provides service information at the bottom of every page.
 
 ## Guidelines
 
-- The Footer consists of a dark blue [Spotlight](/docs/components-containers-spotlight--docs) and a [Page Menu](/docs/components-navigation-page-menu--docs).
+- The Footer consists of two sections: a full-width blue area and a [Page Menu](/docs/components-navigation-page-menu--docs) below it.
   It must be used on all websites for the City of Amsterdam.
-- For applications, only the Page Menu can be sufficient.
 - The Footer is the same on every page of the application.
+- For applications, only the Page Menu can be sufficient.
```

Comment:
By describing it like this, it's easy to mess up the semantics. We currently do this in the Amopis prototype, the Page Menu isn't wrapped in a `footer`.

A bit of a side note, but: I made [a ticket to incorporate the PageMenu in Footer](https://gemeente-amsterdam.atlassian.net/browse/DES-1110), that should help with this.

Edit: [I see you found the same issue](https://gemeente-amsterdam.atlassian.net/browse/DES-1067)

Reply:
This line was only moved. Should we clarify the docs now or when we pick up the tickets you mentioned?

Reply:
Maybe we should do it now, to hopefully prevent people from making this mistake.